provision/acc_provision/acc_provision.py

```diff
@@ -1280,6 +1286,9 @@ def parse_args(show_help):
     parser.add_argument(
         '--disable-multus', default='true', metavar='disable_multus',
         help='true/false to disable/enable multus in cluster')
+    parser.add_argument(
+        '--operator-mode', default=False,
```

Comment:
Since this operator-mode flag is to be used only by the ansible operator internally, maybe it makes more sense to keep it False by default for our testcases?

Reply:
I am not suggesting that it be changed. I am suggesting a comment be added in the code to clarify what is the purpose of this, and I have also suggested what that comment should be (so that its clear what my suggestion is). I had not put the comment in quotes earlier, now I have (in case that was the source of confusion).

Reply:
I meant to post this reply to your comment before this, corrected it now

Yes I agree we need the description.